mqtt/README.md

```diff
@@ -19,7 +19,7 @@
   "hardware_serial": "0102030405060708", // In case of LoRaWAN: the DevEUI
   "port": 1,                          // LoRaWAN FPort
   "counter": 2,                       // LoRaWAN frame counter
-  "is_retry": false,                  // Is set to true if this message is a retry (you could also detect this from the counter)
+  "is_retry": false,                  // Is set to true if this message is a retry (you could also detect this from the counter) - could be left out
```

Comment:
Below is already a line saying:
_Note: Some values may be omitted if they are `null`, `false`, `""` or `0`._